src/components/Admin/Trainer/Settings.js

```diff
@@ -0,0 +1,122 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Input from './Input';
+
+export default class Settings extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			advanced: false
+		};
+	}
+
+	toggleAdvanced = () => {
+		this.setState({advanced: !this.state.advanced});
+	};
+
+	render() {
+		return (
+			<div className="col">
+				<div className="formGroup">
+					<label>
+						<input
+							type="checkbox"
+							onChange={this.toggleAdvanced}
+							checked={this.state.advanced}
+						/>Show advanced settings
+					</label>
+				</div>
+
+				<Input
```

Comment:
Can currently enter any string into these inputs - best to use something like `<input type="number" min="0"...` to make the parsing of values safer